pkgs/development/libraries/librealsense/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, cmake, pkgconfig, libusb }:
+
+stdenv.mkDerivation rec {
+  name = "librealsense-${version}";
+  version = "2.11.0";
+
+  src = fetchurl {
+    url = "https://github.com/IntelRealSense/librealsense/archive/v${version}.tar.gz";
+    sha256 = "1q4yvwnpr2lslq1a33h4zr8g66zz49fsl3pzdwp52gfb6nwyb44a";
+  };
+
+  nativeBuildInputs = [
+    libusb
+    pkgconfig
+    cmake
```

Comment:
When using `cmake`, it is a good idea to add `ninja` as well – it will run slightly faster than the Make backend.